From: Wolfram Quester <wolfi@sigxcpu.org>
Date: Mon, 2 Aug 2010 09:34:15 +0200
Subject: [PATCH] apply patch for correct envelope-from and CC: handling

---
 backend/imap.php |   21 +++++++++++++++++++--
 1 files changed, 19 insertions(+), 2 deletions(-)

diff --git a/backend/imap.php b/backend/imap.php
index c51dc0f..a11aec1 100644
--- a/backend/imap.php
+++ b/backend/imap.php
@@ -108,6 +108,7 @@ class BackendIMAP extends BackendDiff {
         $headers = "";
         $forward_h_ct = "";
         $forward_h_cte = "";
+        $envelopefrom = NULL;
 
         $use_orgbody = false;
 
@@ -155,6 +156,7 @@ class BackendIMAP extends BackendDiff {
                 if      (IMAP_DEFAULTFROM == 'username') $v = $this->_username;
                 else if (IMAP_DEFAULTFROM == 'domain')   $v = $this->_domain;
                 else $v = $this->_username . IMAP_DEFAULTFROM;
+                $envelopefrom = "-f$v";
             }
 
             // check if "Return-Path"-header is set
@@ -179,6 +181,7 @@ class BackendIMAP extends BackendDiff {
             else $v = $this->_username . IMAP_DEFAULTFROM;
             if ($headers) $headers .= "\n";
             $headers .= 'From: '.$v;
+            $envelopefrom = "-f$v";
         }
 
         // set "Return-Path" header if not set on the device
@@ -237,7 +240,16 @@ class BackendIMAP extends BackendDiff {
         //debugLog("IMAP-SendMail: subject: {$message->headers["subject"]}");
         //debugLog("IMAP-SendMail: body: $body");
 
-        $send =  @imap_mail ( $toaddr, $message->headers["subject"], $body, $headers, $ccaddr, $bccaddr);
+        $Cheaders = '';
+        if (!empty($ccaddr)) {
+            $Cheaders .= "Cc: $ccaddr\n";
+        }
+        if (!empty($bccaddr)) {
+            $Cheaders .= "Bcc: $bccaddr\n";
+        }
+        $Cheaders .= $headers;
+
+        $send =  @mail ( $toaddr, $message->headers["subject"], $body, $Cheaders, $envelopefrom );
 
         // email sent?
         if (!$send) {
@@ -248,7 +260,12 @@ class BackendIMAP extends BackendDiff {
         // build complete headers
         $cheaders  = "To: " . $toaddr. "\n";
         $cheaders .= "Subject: " . $message->headers["subject"] . "\n";
-        $cheaders .= "Cc: " . $ccaddr . "\n";
+        if (!empty($ccaddr)) {
+            $cheaders .= "Cc: $ccaddr\n";
+        }
+        if (!empty($bccaddr)) {
+            $cheaders .= "Bcc: $bccaddr\n";
+        }
         $cheaders .= $headers;
 
         $asf = false;
-- 
