From: Wolfram Quester <wolfi@sigxcpu.org>
Date: Wed, 11 Aug 2010 16:02:36 +0200
Subject: [PATCH] convert line endings in mails correctly

---
 backend/imap.php |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/backend/imap.php b/backend/imap.php
index a11aec1..8e26f75 100644
--- a/backend/imap.php
+++ b/backend/imap.php
@@ -249,6 +249,8 @@ class BackendIMAP extends BackendDiff {
         }
         $Cheaders .= $headers;
 
+        $body = str_replace("\r\n", "\n", $body);
+
         $send =  @mail ( $toaddr, $message->headers["subject"], $body, $Cheaders, $envelopefrom );
 
         // email sent?
@@ -920,4 +922,4 @@ class BackendIMAP extends BackendDiff {
 
 };
 
-?>
\ No newline at end of file
+?>
-- 
