From: Wolfram Quester <wolfi@sigxcpu.org>
Date: Sun, 17 Apr 2011 15:49:03 +0200
Subject: set correct d-push version

---
 version.php |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/version.php b/version.php
index c1964cb..436fd75 100644
--- a/version.php
+++ b/version.php
@@ -41,5 +41,5 @@
 * Consult LICENSE file for details
 ************************************************/
 
-$zpush_version = "1.5.10-1262";
-?>
\ No newline at end of file
+$zpush_version = "1.5.10-1 (d-push)";
+?>
