From: Wolfram Quester <wolfi@sigxcpu.org>
Date: Sun, 17 Apr 2011 16:05:40 +0200
Subject: set IMAP as default backend

---
 config.php |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/config.php b/config.php
index 65ff02c..8e1f8b8 100644
--- a/config.php
+++ b/config.php
@@ -184,8 +184,8 @@
 /**********************************************************************************
  *  Backend settings
  */
-    // the backend data provider
-    define('BACKEND_PROVIDER', '');
+    // The data providers that we are using (see configuration below)
+    define('BACKEND_PROVIDER', "BackendIMAP");
 
 /**********************************************************************************
  *  Search provider settings
@@ -255,4 +255,4 @@
 */
     );
 
-?>
\ No newline at end of file
+?>
