From: Wolfram Quester <wolfi@sigxcpu.org>
Date: Sun, 17 Apr 2011 16:05:40 +0200
Subject: set IMAP as default backend

---
 config.php |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/config.php b/config.php
index ba65606..4abdd7a 100644
--- a/config.php
+++ b/config.php
@@ -92,7 +92,7 @@
     define('SYNC_FILTERTIME_MAX', SYNC_FILTERTYPE_ALL);
 
     // The data providers that we are using (see configuration below)
-    $BACKEND_PROVIDER = "BackendICS";
+    $BACKEND_PROVIDER = "BackendIMAP";
 
     // ************************
     //  BackendICS settings
@@ -142,4 +142,4 @@
     // use 'SearchLDAP' to search in a LDAP directory (see backend/searchldap/config.php)
     define('SEARCH_PROVIDER', '');
 
-?>
\ No newline at end of file
+?>
-- 
