From: Wolfram Quester <wolfi@sigxcpu.org>
Date: Sun, 17 Apr 2011 16:05:40 +0200
Subject: set IMAP as default backend

---
 config.php |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/config.php b/config.php
index 0838b4a..45df41d 100644
--- a/config.php
+++ b/config.php
@@ -97,7 +97,7 @@
     define('PING_INTERVAL', 30);
 
     // The data providers that we are using (see configuration below)
-    $BACKEND_PROVIDER = "BackendICS";
+    $BACKEND_PROVIDER = "BackendIMAP";
 
     // ************************
     //  BackendICS settings
@@ -147,4 +147,4 @@
     // use 'SearchLDAP' to search in a LDAP directory (see backend/searchldap/config.php)
     define('SEARCH_PROVIDER', '');
 
-?>
\ No newline at end of file
+?>
-- 
