From: Wolfram Quester <wolfi@sigxcpu.org>
Date: Sun, 17 Apr 2011 16:05:40 +0200
Subject: set proper default config

---
 config.php |   15 ++++++++++-----
 1 file changed, 10 insertions(+), 5 deletions(-)

diff --git a/config.php b/config.php
index 8e1f8b8..d3630fb 100644
--- a/config.php
+++ b/config.php
@@ -66,7 +66,7 @@
 /**********************************************************************************
  *  Default FileStateMachine settings
  */
-    define('STATE_DIR', '/var/lib/z-push/');
+    define('STATE_DIR', '/var/lib/d-push/state/');
 
 
 /**********************************************************************************
@@ -86,9 +86,14 @@
  *  ones, e.g. setting to LOGLEVEL_DEBUG will also output LOGLEVEL_FATAL, LOGLEVEL_ERROR,
  *  LOGLEVEL_WARN and LOGLEVEL_INFO level entries.
  */
-    define('LOGFILEDIR', '/var/log/z-push/');
-    define('LOGFILE', LOGFILEDIR . 'z-push.log');
-    define('LOGERRORFILE', LOGFILEDIR . 'z-push-error.log');
+    define('LOGFILEDIR', '/var/log/d-push/');
+    define('LOGFILE', LOGFILEDIR . 'd-push.log');
+    define('LOGERRORFILE', LOGFILEDIR . 'd-push-error.log');
+    // Possible Loglevels are:
+    // LOGLEVEL_OFF, LOGLEVEL_FATAL, LOGLEVEL_ERROR, LOGLEVEL_WARN,
+    // LOGLEVEL_INFO, LOGLEVEL_DEBUG, LOGLEVEL_WBXML, LOGLEVEL_DEVICEID,
+    // LOGLEVEL_WBXMLSTACK, LOGLEVEL_ALL
+    // see /usr/share/d-push/lib/core/zpushdefs.php for more info
     define('LOGLEVEL', LOGLEVEL_INFO);
     define('LOGAUTHFAIL', false);
 
@@ -229,7 +234,7 @@
  *  - on Zarafa systems use backend/zarafa/listfolders.php script to get a list
  *    of available folders
  *
- *  - all Z-Push users must have full writing permissions (secretary rights) so
+ *  - all D-Push users must have full writing permissions (secretary rights) so
  *    the configured folders can be synchronized to the mobile
  *
  *  - this feature is only partly suitable for multi-tenancy environments,
