From: Wolfram Quester <wolfi@sigxcpu.org>
Date: Sat, 24 Mar 2012 14:33:08 +0100
Subject: Use mail() in sample config

---
 config.php |    6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/config.php b/config.php
index 45df41d..95aa871 100644
--- a/config.php
+++ b/config.php
@@ -127,8 +127,10 @@
     define('IMAP_SENTFOLDER', '');
     // forward messages inline (default off - as attachment)
     define('IMAP_INLINE_FORWARD', false);
-    // use imap_mail() to send emails (default) - off uses mail()
-    define('IMAP_USE_IMAPMAIL', true);
+    // don't use imap_mail() to send emails.
+    // true (default, uses imap_mail, which is broken - false uses mail(),
+    // which handles cc and from in a more sane way)
+    define('IMAP_USE_IMAPMAIL', false);
 
 
     // ************************
