From: Wolfram Quester <wolfi@sigxcpu.org>
Date: Sun, 17 Apr 2011 16:05:40 +0200
Subject: set IMAP as default backend

---
 config.php |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/config.php b/config.php
index 7ce0d90..f6533cb 100644
--- a/config.php
+++ b/config.php
@@ -122,8 +122,7 @@
  *  Backend settings
  */
     // The data providers that we are using (see configuration below)
-    define('BACKEND_PROVIDER', "BackendZarafa");
-
+    define('BACKEND_PROVIDER', "BackendIMAP");
 
     // ************************
     //  BackendZarafa settings
