From: Wolfram Quester <wolfi@sigxcpu.org>
Date: Sat, 13 Apr 2013 18:02:42 +0200
Subject: adapt path variable in migration tool

---
 tools/migrate-2.0.x-2.1.0.php |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: z-push/tools/migrate-2.0.x-2.1.0.php
===================================================================
--- z-push.orig/tools/migrate-2.0.x-2.1.0.php	2017-01-04 14:11:43.903584491 +0100
+++ z-push/tools/migrate-2.0.x-2.1.0.php	2017-01-04 14:11:58.127626735 +0100
@@ -26,7 +26,7 @@
 ************************************************/
 
 // Please adjust to match your z-push installation directory, usually /usr/share/z-push
-define('ZPUSH_BASE_PATH', "../src");
+define('ZPUSH_BASE_PATH', "/usr/share/z-push");
 
 
 
