From: Wolfram Quester <wolfi@sigxcpu.org>
Date: Sat, 13 Apr 2013 18:02:42 +0200
Subject: adapt path variable in migration tool

---
 tools/migrate-2.0.x-2.1.0.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/migrate-2.0.x-2.1.0.php b/tools/migrate-2.0.x-2.1.0.php
index 8d2f0a9..3e5327d 100755
--- a/tools/migrate-2.0.x-2.1.0.php
+++ b/tools/migrate-2.0.x-2.1.0.php
@@ -26,7 +26,7 @@
 ************************************************/
 
 // Please adjust to match your z-push installation directory, usually /usr/share/z-push
-define('ZPUSH_BASE_PATH', "../src");
+define('ZPUSH_BASE_PATH', "/usr/share/z-push");
 
 
 
