Index: z-push/src/config.php
===================================================================
--- z-push.orig/src/config.php	2017-01-04 10:14:11.428109787 +0100
+++ z-push/src/config.php	2017-01-04 13:57:40.905080329 +0100
@@ -61,7 +61,7 @@
  *           State migration script is available, more informations: https://wiki.z-hub.io/x/xIAa
  */
     define('STATE_MACHINE', 'FILE');
-    define('STATE_DIR', '/var/lib/z-push/');
+    define('STATE_DIR', '/var/lib/z-push/state/');
 
 /**********************************************************************************
  *  IPC - InterProcessCommunication
