Index: z-push-packaging/tools/gab2contacts/gab2contacts.php
===================================================================
--- z-push-packaging.orig/tools/gab2contacts/gab2contacts.php
+++ z-push-packaging/tools/gab2contacts/gab2contacts.php
@@ -26,7 +26,7 @@
 
 // Path to the Z-Push directory relative to the gab2contacts script.
 // The path set by default is as required for a GIT checkout.
-define('PATH_TO_ZPUSH', '../../src/');
+define('PATH_TO_ZPUSH', '/usr/share/z-push/');
 
 /************************************************
  * MAIN
