Index: z-push-packaging/config/apache2/z-push.conf
===================================================================
--- z-push-packaging.orig/config/apache2/z-push.conf	2017-03-23 13:19:11.912399823 +0100
+++ z-push-packaging/config/apache2/z-push.conf	2017-03-23 13:24:14.124947373 +0100
@@ -23,20 +23,13 @@
     # Security
     # Don't allow .htaccess Overrides, disallow access to files
     AllowOverride none
-    <IfModule !mod_authz_core.c>
-        Order allow,deny
-        allow from all
-    </IfModule>
     <IfModule mod_authz_core.c>
         Require all granted
     </IfModule> 
 
     <Files "config.php">
-      <IfModule !mod_authz_core.c>
-        Deny from All
-      </IfModule>
       <IfModule mod_authz_core.c>
         Require all denied
       </IfModule>
     </Files>
-</Directory>
\ No newline at end of file
+</Directory>
